vignettes/adverse_events.Rmd

```diff
@@ -0,0 +1,109 @@
+---
+title: "Adverse Events (AE)"
+output: 
+  rmarkdown::html_vignette:
+    number_sections: true
+vignette: >
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+date: "2025-01-30"
+---
+
+<!-- https://oncostat.github.io/grstat/articles/adverse_events_sas.html -->
+
+# Introduction
+
+This vignette demonstrates how to implement adverse event (AE) tables in R using the `grstat` package. AE tables are a standard approach in clinical research for assessing event severity (grade) and specifying the exact AE terms.
+
+In this article, three types of AE tables are presented:
+
+-   Maximum AE grade per patient (for both single-arm and two-arm trials)
+
+-   Maximum AE grade per patient by AE term (for both single-arm and two-arm trials)
```

Comment:
Focus on the 2 functions, rather than the 3 types.
Maybe you can then explain that there are 3 variants and explain what they are.